<?xml version="1.0" encoding="UTF-8"?>
<CustomMetadata xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
    <label>Record12</label>
    <protected>false</protected>
    <values>
        <field>Description__c</field>
        <value xsi:type="xsd:string">Request has been rejected.  The following may trigger this error: - Referential Match reference file schema validation failure during upload (response will have URL with audit file). - Input file could not be processed.</value>
    </values>
    <values>
        <field>ErrorMessage__c</field>
        <value xsi:nil="true"/>
    </values>
    <values>
        <field>HTTPStatusCode__c</field>
        <value xsi:nil="true"/>
    </values>
    <values>
        <field>IsDataError__c</field>
        <value xsi:type="xsd:boolean">false</value>
    </values>
    <values>
        <field>errorCode__c</field>
        <value xsi:type="xsd:string">60105</value>
    </values>
    <values>
        <field>product__c</field>
        <value xsi:nil="true"/>
    </values>
</CustomMetadata>
